modules/specialized-guides/pages/salt/salt-formula-virtualization-host.adoc

```diff
@@ -0,0 +1,25 @@
+[[virt-host-formula]]
+= Virtualization Host Formula
+
+The VsFTPd formula is used to configure the FTP service on the branch server.
```

Comment:
```suggestion
The [literal``VsFTPd`` formula is used to configure the FTP service on the branch server.
```

Reply:
This was actually a paste error from some other file.  Fixed separately.